### ISSUE TITLE
Told I'm missing dwmapi.dll

### ISSUE THREAD

**Issue by eyalroz** [unetbootin/unetbootin#304]
I downloaded UNetbootin for Windows from [here](https://unetbootin.github.io/), and am trying to use it on a machine running Windows XP. I get the error:

> This application has failed to start because dwmapi.dll was not found. Re-installing the application may fix the problem.

Well, there was no installing: The download is an `.exe` file (`unetbootyin-windows-702.exe`).

Am I doing something wrong or is it a bug with UNetbootin?

**Comment by JonnyTech** [comment]
Use the last recommended version for XP compatibilty, not the latest release.

**Comment by eyalroz** [comment]
@JonnyTech : Certainly, but -"Windows XP" doesn't appear anywhere on the page. How do I (or any user) get to that version?

**Comment by JonnyTech** [comment]
Try v6.81 as that was the last version to use QT4:

https://github.com/unetbootin/unetbootin/releases/download/681/unetbootin-windows-681.exe

Newer builds use QT5 which does not support XP.

**Comment by eyalroz** [comment]
@JonnyTech : Thanks. I would also asked that this be mentioned in the FAQ, and/or that the download "buttons" on the main site mention that Vista or later is required.

**Comment by mobluse** [comment]
The same problem exists on PortableApps where it says it works with Windows XP, but then it doesn't start:
https://portableapps.com/apps/utilities/unetbootin-portable
https://portableapps.com/downloading/?a=UNetbootinPortable&s=s&p=&d=pa&n=UNetbootin%20Portable&f=UNetbootinPortable_702.paf.exe

"System Requirements: Windows XP, Vista, 7, 8, 10, 11"

I upgraded my PortableApps and then it automatically upgraded UNetbootin to a non working version and it seems to be difficult to go back because I do not have the last working paf file for UNetbootin. You should perhaps make a legacy release for XP on PortableApps as some other projects have done, so that one is not prompted to upgrade to a non working version. The legacy apps for XP can be seen when you search for apps using PortableApps on the XP computer.
